docs/3.0.0-beta.x/advanced/traceability.md

```diff
@@ -0,0 +1,44 @@
+# Traceability
```

Comment:
Find another word @davidkartuzinski We need your help for that

Reply:
Usage metrics